installpolicy.js

```diff
@@ -348,9 +348,6 @@ var getPolicyId = function(policyname) {
     logger.info(`Starting getPolicyId function. Get Policy Id by Name: ${policyname} `);
   }
 
-  var getPolicyFound = true;
-  var getPolicyList = `https://localhost/mgmt/tm/asm/policies?ver=${ver}`;
-
   return new Promise(function(resolve, reject) {
     // Create the algorithm.
     const algo = crypto.createHash('md5');
```

Comment:
The idea make sense. 
Trying to add "crypto" module, I receive error:

[root@host-10-240-58-112:Active:Standalone] node_modules # npm install crypto
npm WARN deprecated crypto@1.0.1: This package is no longer supported. It's now a built-in Node module. If you've depended on crypto, you should switch to the one that's built-in.
/var/config/rest/iapps/InstallPolicy
└── crypto@1.0.1

npm WARN enoent ENOENT: no such file or directory, open '/var/config/rest/iapps/InstallPolicy/package.json'
npm WARN InstallPolicy No description
npm WARN InstallPolicy No repository field.
npm WARN InstallPolicy No README data
npm WARN InstallPolicy No license field.

So I moved to "md5" module but recieve different string then the ASM generate:
var policyIDmd5 = md5(policyname);
    logger.info("policy Id by MD5 hash is: " + policyIDmd5);

Wed, 06 Jun 2018 11:51:57 GMT - info: policy Id by MD5 hash is: ac44f0eef4dfb40dceccdcff43f85b2a
Wed, 06 Jun 2018 11:51:58 GMT - info: Get Policy ID by Name: Declarative_API_Policy Completed, Policy Id: Z3WrvZivWUyJaURQnLyb2w